CMakeLists.txt

```diff
@@ -137,7 +147,9 @@ install (FILES src/pipeline.h DESTINATION include/nanomsg)
 install (FILES src/survey.h DESTINATION include/nanomsg)
 install (FILES src/bus.h DESTINATION include/nanomsg)
 
-install (TARGETS nanocat RUNTIME DESTINATION bin)
+if (NN_TOOLS)
```

Comment:
You could have both. I am currently bundling nanomsg inside a project, and passing `-DNN_TOOLS=OFF` to disable all tools would be more convenient than specifying a list of `-DNN_ENABLE_XXX=OFF` (and update it if any tool gets added in the future). I agree that this is very far fetched since there is only nanocat, though.

On a complexity standpoint, adding a `NN_TOOLS` option with specific `NN_ENABLE_XXX` options isn't doing much: You'd just have to specify the default value of every `NN_ENABLE` option with `${NN_TOOLS}`.